sympy/printing/tests/test_latex.py

```diff
@@ -2174,6 +2175,15 @@ def test_KroneckerProduct_printing():
     assert latex(KroneckerProduct(A, B)) == r'A \otimes B'
 
 
+def test_TransferFunction_printing():
+    tf1 = TransferFunction(x - 1, x + 1, x)
+    assert latex(tf1) == "TransferFunction\\left(x - 1, x + 1, x\\right)"
```

Comment:
The latex form should not include "TransferFunction" is should likely print the same as a Rational.